app/controllers/course/assessment/question/scribing_controller.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+class Course::Assessment::Question::ScribingController < \
+  Course::Assessment::QuestionsController
+  load_and_authorize_resource :scribing_question,
+                              class: Course::Assessment::Question::Scribing,
+                              through: :assessment, parent: false
+
+  def new
+    @template = 'course/assessment/question/scribing/new.json.jbuilder'
```

Comment:
Not sure if this is still necessary. The programming questions use redirection and there are async code, that's why the backend renders the template and displays the the validation error.

You can check the implementation in survey questions, it has full CRUD: https://github.com/Coursemology/coursemology2/blob/master/app/controllers/course/survey/questions_controller.rb